docs/content/configuration-language.mdx

```diff
@@ -283,15 +284,18 @@ For example, let's take a closer look at the `team` type.
     },
     metadata: {
       relations: {
-        member: { directly_related_user_types: [{ type: 'user' }, { type: 'team', relation: 'member' }] },
+        member: { directly_related_user_types: [{ type: 'user' }, { type: 'user:*'}, { type: 'team', relation: 'member' }] },
       },
     },
-  }}
+  }} skipVersion={true}
 />
 
 This `team` _<ProductConcept section="what-is-a-type-definition" linkName="type definition" />_ defines all the _<ProductConcept section="what-is-a-relation" linkName="relations" />_ that _<ProductConcept section="what-is-a-user" linkName="users" />_ can have with an _<ProductConcept section="what-is-an-object" linkName="object" />_ of _type_ `team`. In this case the _relation_ is: `member`.
 
-Due to the direct relationship keyword (`self`) being used, a user in the system can have a **<ProductConcept section="what-are-direct-and-implied-relationships" linkName="direct relationship" />** with the `team` type as a `member`.
+Due to the direct relationship type restrictions (`[user, team#member]`) being used, a user in the system can have a **<ProductConcept section="what-are-direct-and-implied-relationships" linkName="direct relationship" />** with the `team` type as a `member` for users of 
+- type `user`
+- <ProductConcept section="what-is-type-bound-public-access" linkName="type bound public access" /> of type `user`
```

Comment:
Maybe `public access bound to the user type` when used here in this sentence, or `user type bound public access`

Reply:
I will use user type bound public access.  I prefer to keep `type bound public access` together as a term.